tests/qat/test_qasm.py

```diff
@@ -603,6 +604,18 @@ def test_sech(self):
         assert np.isclose(pulses[1].amp, 0.5)
         assert np.isclose(pulses[1].std_dev, 20e-9)
 
+    @pytest.mark.parametrize(
+        "qasm", get_default_qasm3_gate_qasms(), ids=lambda val: val.split("\n")[-2]
+    )
+    def test_default_gates(self, qasm):
```

Comment:
Would it be possible to have a test with an QASM3 script that uses multiple of the problematic gate(s)?